server/src/routes/swagger.ts

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+/**
+ * This file is used as a base for the swagger complition in the index.js file of routes.
+ */
+
+export const definition: object = {
```

Comment:
Is there a more specific type available for this in an `@types` package?

Reply:
`swaggerJSDoc.OAS3Definition` seems to be the right type.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/swagger-jsdoc/index.d.ts

Reply:
Changed typing